app/views/products/show.html.erb

```diff
@@ -1,15 +1,85 @@
-<div class="container p-3">
-<h1> <%= @product.name %></h1>
-<div class="product-card d-flex justify-content-between">
-<img src=" <%= @product.picture %>"/>
-<div class="product-info p-3">
-<p><strong>About</strong></p>
-<p> <%= @product.description %><p>
-<p><strong>Price</strong></p>
-<p> <%= @product.price %>$<p>
-<div class="btn btn-primary">Rent</div>
-</div>
-</div>
-</div>
-
-
+<nav class="flex-nav">
```

Comment:
just a heads up, this may change as it would needed to be according to the wireframe. otherwise for now, its good